### ISSUE TITLE
Anyone Want to help with VOIP?

### ISSUE THREAD

**Issue by SilverStrings024** [leogomezz4t/PyTextNow_API#38]
In the branch `unstable-pre-release-2.1.0` we're implementing VOIP.
If you have the ability to work with VOIP and want to, we'd love to see you submit a PR or two to help get it stable. If you do choose to work on the VOIP, please tag me (SilverStrings024) and ask to be brought into the Discord so we can discuss your ideas and changes with the team and implement it in a way that keeps the package stable.

Note that in the `unstable-pre-release-2.0.0` branch, we're completely removing **ALL** references to VOIP so, if you want to work on it, you'll need to use `unstable-pre-release-2.1.0`

Thank you for any help!

**Comment by anask1234** [comment]
yeah sure umm still need help? btw im a beginner at this stuff so not sure i can help too much.
